lib/index.js

```diff
@@ -31,6 +45,57 @@ internals.implementation = function (server, options) {
                 return reply(Boom.unauthorized(null, 'Access Token'));
             }
     
+            var parts = parseCookies(authorization);
```

Comment:
Hi guys! Thanks for the contribution! 
I'm really under a run on my job right now and gonna give a look at this requests in the weekend.